### PR TITLE
Test structure

### PR DESCRIPTION
**Pull request by Chava888**

### CODE REVIEW

File: src/pages/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import App from '../App'
+import Home from './Home'
+
+Enzyme.configure({adapter: new Adapter()})
+
+describe("Check For Home Component...",() => {
+  it("As a developer, I have test coverage on my Home component.", () => {
+    const renderedApp = shallow(<App/>)
```
Comment (WalkerTexas-coder) [inline src/pages/Home.test.js]: The Home test should be testing the Home Component not the App component. Feel free to move this test into the App.test.js.

**Review by WalkerTexas-coder — COMMENTED** [review]
One comment

File: src/pages/Home.test.js (hunk 1 above)
Comment (Chava888) [reply src/pages/Home.test.js]: We made changes to the Home.test.js to actually reference the home component.

**Review by WalkerTexas-coder — APPROVED** [review]
Excellent work!

**Pull request merged**